src/test/java/mil/dds/anet/test/resources/AttachmentResourceTest.java

```diff
@@ -34,181 +33,193 @@ public class AttachmentResourceTest extends AbstractResourceTest {
   @Test
   public void testCreateAttachment()
       throws GraphQLRequestPreparationException, GraphQLRequestExecutionException {
-    final ReportInput testReportInput =
-        ReportInput.builder().withIntent("a test report created by testCreateAttachment")
-            .withReportPeople(
-                getReportPeopleInput(Collections.singletonList(personToReportAuthor(admin))))
-            .build();
-    final Report testReport = adminMutationExecutor.createReport(REPORT_FIELDS, testReportInput);
-    assertThat(testReport).isNotNull();
-    assertThat(testReport.getUuid()).isNotNull();
-
-    // Attach attachment to test report
-    final AttachmentRelatedObjectInput testAroInput =
-        createAttachmentRelatedObject(ReportDao.TABLE_NAME, testReport.getUuid());
-
-    final AttachmentInput failedAttachmentInput =
-        AttachmentInput.builder().withFileName("testAttachment.jpg")
-            .withDescription("a test attachment created by testCreateAttachment")
-            .withClassification(Classification.UNDEFINED)
-            .withAttachmentRelatedObjects(Collections.singletonList(testAroInput)).build();
-
-    // Fail attachment create with a mimetype that is not allowed
-    failAttachmentCreate(adminMutationExecutor, failedAttachmentInput);
-
     final Map<String, Object> attachmentSettings = (Map<String, Object>) AnetObjectEngine
         .getConfiguration().getDictionaryEntry("fields.attachment");
-    final var allowedMimeTypes = (List<String>) attachmentSettings.get("mimeTypes");
-    final Boolean userUploadDisabled = (Boolean) attachmentSettings.get("disabled");
-    final String mimeType = allowedMimeTypes.get(0);
-    failedAttachmentInput.setMimeType(mimeType);
-
-    // Fail attachment create with wrong classification
-    failedAttachmentInput.setClassification(Classification.NATO_UNCLASSIFIED);
-    failAttachmentCreate(adminMutationExecutor, failedAttachmentInput);
-
-    if (userUploadDisabled) {
-      // Fail attachment create with any user other than admin
-      final MutationExecutor erinMutationExecutor =
-          getMutationExecutor(getRegularUser().getDomainUsername());
-      failAttachmentCreate(erinMutationExecutor, failedAttachmentInput);
+    final Boolean attachmentDisabled = (Boolean) attachmentSettings.get("featureDisabled");
+
+    if (!attachmentDisabled) {
```

Comment:
Why is this now a conditional check? We know attachments are enabled in the dictionary used for testing, so the checks should be unconditional.